resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/shuffle-service/Dockerfile

```diff
@@ -15,25 +15,12 @@
 # limitations under the License.
 #
 
-FROM openjdk:8-alpine
+FROM spark-base
```

Comment:
In the past I have used a literal `snapshot` tag, but I think `latest` leverages conventions that some image pull policies have, where they always look to pull the latest image from the registry